stream-chat-android-client/src/main/java/io/getstream/chat/android/client/extensions/ChannelExtension.kt

```diff
@@ -15,5 +15,5 @@ public fun Channel.isAnonymousChannel(): Boolean = id.isAnonymousChannelId()
 public fun Channel.isMutedFor(user: User): Boolean = user.channelMutes.any { mute -> mute.channel.cid == cid }
 
 @InternalStreamChatApi
-public fun Channel.getUsersExcludingCurrent(excludeUserId: String = ChatClient.instance().getCurrentUser()?.id ?: ""): List<User> =
-    members.map { it.user }.filterNot { it.id == excludeUserId }
+public fun Channel.getUsersExcludingCurrent(): List<User> =
+    members.map { it.user }.filterNot { it.id == ChatClient.instance().getCurrentUser()?.id ?: "" }
```

Comment:
We do a bunch of comparisons to the empty string here in some cases, which is not something we want to do. How about this instead?

```suggestion
public fun Channel.getUsersExcludingCurrent(): List<User> {
    val users = members.map { it.user }

    val currentUserId = ChatClient.instance().getCurrentUser()?.id
    return if (currentUserId != null) {
        users.filterNot { it.id == currentUserId }
    } else {
        users
    }
}
```